### PR TITLE
drop the pointer from the job typedef

### PR DESCRIPTION
**Pull request by ysmolsky**
This patch is a hell of a (Job *).

Updates #458

### CODE REVIEW

File: job.c [hunk 1]
```diff
@@ -176,24 +176,24 @@ job_pri_less(void *ja, void *jb)
...
 
     if (!j) return NULL;
 
-    n = malloc(sizeof(struct job) + j->r.body_size);
-    if (!n) return twarnx("OOM"), (job) 0;
+    n = malloc(sizeof(struct Job) + j->r.body_size);
+    if (!n) return twarnx("OOM"), (Job *) 0;
 
-    memcpy(n, j, sizeof(struct job) + j->r.body_size);
+    memcpy(n, j, sizeof(struct Job) + j->r.body_size);
```
Comment (kr) [inline job.c]: Optionally I think these could say `sizeof(Job)` now if you wanted.

File: main.c [hunk 2]
```diff
@@ -99,7 +99,7 @@ main(int argc, char **argv)
             exit(10);
         }
 
-        struct job list = {.prev=NULL, .next=NULL};
+        struct Job list = {.prev=NULL, .next=NULL};
```
Comment (kr) [inline main.c]: Same for declarations, I think this could be `Job list = ...` now. (But no big deal either way IMO.)

File: job.c (hunk 1 above)
Comment (ysmolsky) [reply job.c]: _Job's done._

File: main.c (hunk 2 above)
Comment (ysmolsky) [reply main.c]: _Job's done._

**Comment by ysmolsky** [comment]
I am sorry for the jokes. I just could not restrain myself. 🏥

**Pull request merged**